Completions api legacy (#2074)
Kong/developer.konghq.com: app/_includes/plugins/ai-proxy/overview.md

{{ page.description | liquify }}

{{ plugin }} plugin accepts requests in one of a few defined and standardized OpenAI formats, translates them to the configured target format, and then transforms the response back into a standard format.

{% new_in 3.10 %} To use {{ plugin }} with non-OpenAI format without conversion, see [section below](./#supported-native-llm-formats) for more details.

## Overview of capabilities

{{ plugin }} plugin supports capabilities across batch processing, multimodal embeddings, agents, audio, image, streaming, and more, spanning multiple providers:

For {{ site.base_gateway }} versions 3.6 or earlier:

* **Chat Completions APIs**: Multi-turn conversations with system/user/assistant roles.

> * **Completions API**: Generates free-form text from a prompt. This endpoint has been marked as legacy by OpenAI and is not recommended for new applications.

Reviewer: ```suggestion
* **Completions API**: Generates free-form text from a prompt. **OpenAI has marked this endpoint as legacy and recommends using the Chat API for new applications.**
```

{% include plugins/ai-proxy/tables/supported-providers.html providers=providers %}

> Using following providers are supported by the legacy Completions API:

Reviewer: ```suggestion
The following providers are supported by the legacy Completions API:
```
